packages/airnode-examples/scripts/choose-integration.ts

```diff
@@ -94,7 +95,35 @@ const chooseIntegration = async (): Promise<IntegrationInfo> => {
   return response as IntegrationInfo;
 };
 
+/**
+ * If git is installed, check if a tag is checked out
+ */
+const checkGitTag = () => {
+  // skip check on Windows
+  if (isWindows()) return;
```

Comment:
People on SO say that it *should* work on windows as well. Maybe we can at least try? But I'm fine with skipping this check on Windows as well.

Reply:
I modified the commands a bit to now also work on Windows